packages/fxa-auth-server/test/scripts/remove-accounts.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+import {
+  retreiveUnverifiedAccounts,
+  cancelSubscriptionsAndDeleteCustomer,
+  issueRefund,
+} from '../../scripts/remove-accounts';
+
+const ROOT_DIR = '../..';
+const cp = require('child_process');
+const util = require('util');
+const path = require('path');
+const execAsync = util.promisify(cp.exec);
+const sinon = require('sinon');
+const { assert } = require('chai');
+
+const cwd = path.resolve(__dirname, ROOT_DIR);
+const execOptions = {
+  cwd,
+  env: {
+    ...process.env,
+    NODE_ENV: 'dev',
+    LOG_LEVEL: 'error',
+    AUTH_FIRESTORE_EMULATOR_HOST: 'localhost:9090',
+  },
+};
+
+describe('scripts/remove-accounts startup', () => {
+  it('does not fail', function () {
```

Comment:
Q: since `execAsync()` returns a promise, should this be an async function?
Or maybe not since nothing is `await`ed.

Reply:
doesnt need to be async since not awaited